test/library/standard/IO/fileReaderLimited.chpl

```diff
@@ -0,0 +1,24 @@
+use IO;
+
+var filename = "fileReaderLimited.txt";
```

Comment:
Could these tests share the same .txt file?

Reply:
Yeah, this and the openreader one could